modules/op-enabling-manual-approval-gate.adoc

```diff
@@ -0,0 +1,51 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-enabling-manual-approval-gate_{context}"]
+= Enabling the manual approval gate controller
+
+To use manual approval tasks, you must first enable the manual approval gate controller.
+
+.Prerequisites
+
+* You installed the {pipelines-title} Operator in your cluster.
+* You are logged on to the cluster using the `oc` command-line utility.
+* You have administrator permissions for the `openshift-pipelines` namespace.
+
+.Procedure
+
+. Create a file named `manual-approval-gate-cr.yaml` with the following manifest for the `ManualApprovalGate` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: ManualApprovalGate
+metadata:
+  name: manual-approval-gate
+spec:
+  targetNamespace: openshift-pipelines
+----
+
+. Apply the `ManualApprovalGate` CR by entering the following command:
++
+[source,terminal]
+----
+$ oc apply -f <manual-approval-gate-cr>.yaml
+----
+
+. Verify that the manual approval gate controller is running by entering the following command:
++
+[source,terminal]
+----
+$ oc get manualapprovalgates.operator.tekton.dev
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   VERSION    READY   REASON
+manual-approval-gate   v0.1.0	    True
```

Comment:
Just an FYI that this is misaligned in both the file and the rendered preview, not sure if that's intentional:
```suggestion
NAME                   VERSION    READY   REASON
manual-approval-gate   v0.1.0	  True
```

Reply:
This is OUTRIGHT SUPER WEIRD. I have already fixed the same issue once. I will now try again.